src/main/resources/cards/221-delta_species.yaml

```
@@ -2596,9 +2596,9 @@ cards:
   rarity: Uncommon
   text: ['This card stays in play when you play it. Discard this card if another Stadium
       card comes into play. If another card with the same name is in play, you can''t
-      play this card.', Each player's basic Energy cards attached to Pokémon that
-      has δ on its card are both their usual Energy type and Metal type but provide
-      only 1 Energy at a time. (Has no effect other than providing Energy.)]
+      play this card.', 'Each player''s basic Energy cards attached to Pokémon that
```

I am incredibly doubtful this is correct. I assume you need to escape the apostrophe.
```suggestion
      play this card.', 'Each player\'s basic Energy cards attached to Pokémon that
```

---

Alternatively, I'd assume using double quotes for this string instead would work.

---

I saw a line right above this card using double quotes, and apparently is the way used here. 

Couldn't find any use of `\'` in the repo through a file search.

---

> In single quoted strings the single quote character can be escaped by prefixing it with another single quote, basically doubling it.
Nevermind, I see this in the YAML documentation.